### ISSUE TITLE
[BUG] Chart Editor is using QWERTY layout no matter what

### ISSUE THREAD

**Issue by Ksempac** [TeamRizu/OutFox#359]
**Please give your StepMania Info**

StepMania Version: Latest Outfox (4.9.8)
Theme: Default

**Describe the problem**

When you go in the editor, you get told that you should use various shortcuts to create and modify the charts, like "A to enter Alter Menu".
However, the keys are actually hardcoded to QWERTY layout. So, with an AZERTY keyboard I need to use Q to enter the Alter Menu.

**Describe what should happen**

Hmm it is a tricky one.

Solution 1: In theory, we can change the help text so that say, the French translation refers to AZERTY keys. However, there is no 1-1 mapping between language and layout. Also no guarantee someone hasn't hand picked a different keyboard layout.

Solution 2: The text shouldn't be changed and the keys should be mapped according to current keyboard layout. However the issue here is that it may spread out some keys that were supposed to be close together (for example N and M) to something not close at all.

I think Solution 2 is the least bad because it is at least intuitive for someone not familiar with the concept of keyboard layouts. People familiar with this concept are also probably used to switching to QWERTY when necessary, so I expect them to do so.

**Comment by JoseVarelaP** [comment]
Yes, this will be a somewhat tricky situation to deal with, given the entire editor (and pretty much all text input related from the game) is currently locked to the QWERTY US layout.

The second solution does seem the least problematic given the editor does have support for changing input via a ini file on preferences (fun fact, it was never documented from who added it back in sm5), so this could be utilized to create default mappings for particular keyboard layouts.

**Comment by Scraticus** [comment]
This will be resolved in the autumn. I need to remove the ASCII forced keybindings.
